README.md

```diff
@@ -56,7 +56,7 @@ object Main extends App {
           .toManagedZIO
           .useForever
```

Comment:
```suggestion
          .useForever
          .exitCode
```

And you can fully remove `foldCauseM ` block with the approach above

Reply:
thanks - I've applied the suggested change to this PR